Week02/Assignment/gmorbioli_01_code.py

```diff
@@ -0,0 +1,107 @@
+import matplotlib.pyplot as plt, matplotlib,numpy as np
+from IPython.display import Image
+from pylab import *
+from scipy.stats import linregress
+from scipy.optimize import curve_fit
+from scipy.integrate import quad
+
+def figure(x, y,xr=None,yr=None,integral = None):
+    '''  Plots a scatter plot using x as the independent variable and y as the 
+    dependent variable; also, if a regression is performed, it plots the regression as well '''
+    
+    fig = plt.figure() #creates the object figure
+
+    axes = fig.add_axes([0.1, 0.12, 0.85, 0.8]) # left, bottom, width, height (range 0 to 1))
+
+    axes.plot(x,y, 'ro') #plots the data
+    axes.set_xlabel('Independent variable / A.U.') #gives name to the x-axis
+    xlim(0, max(x)+2) #sets the limits of the x axis
+    axes.set_ylabel('Dependent variable / A.U.') #gives name to the y-axis
+    ylim(0, max(y)+5) #sets the limits of the y axis
+    axes.set_title(raw_input('Name of plot: ')) #gives a title to the graph
+    
+    # Only show ticks on the left and bottom spines
+    axes.yaxis.set_ticks_position('left')
+    axes.xaxis.set_ticks_position('bottom')
+    
+    if yr !=None:
+        if xr!=None:
+            #if regression data is available, include that in the plot
+            plt.plot(xr,yr)
+            if integral != None:
+                #if this argument is given, fill the area under the curve and include the value of the integral
+                plt.fill_between(xr,yr,color="lightblue", hatch="XX", edgecolor="lightblue", linewidth=0.0)
+                plt.text(xr[len(xr)/4], 1, '%f A.U.' %integral ,size = '25', ) # includes the Area under the curve value
+    
+    matplotlib.rcParams.update({'font.size': 17}) #Changes the font size
+
+    matplotlib.pyplot.show() # shows the figure
```

Comment:
Since you've `import matplotlib.pyplot as plt` above, you can just use `plt.show()` here.